docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.puml

```diff
@@ -2,7 +2,18 @@
 
 package edu.oakland.test.admin {
     class LocationDataPointTests {
-        
+        + void validLatInIsLngOut()
```

Comment:
I think this should say "validLatInIs**Lat**Out()"

Reply:
Good catch! Thank you!